app/src/main/resources/application.properties

```diff
@@ -35,3 +32,38 @@ quarkus.index-dependency.jaxrs.artifact-id=jboss-jaxrs-api_2.1_spec
 %dev.registry.kafka.snapshot-producer.acks=all
 %dev.registry.kafka.snapshot.requests=1000
 %dev.registry.kafka.snapshot-consumer.bootstrap.servers=localhost:9092
+
+# === Prod profile - see README
+
+# Log
+%prod.quarkus.log.level=INFO
+%prod.quarkus.log.console.enable=true
+
+# JPA
+%prod.quarkus.datasource.driver=org.postgresql.Driver
+%prod.quarkus.datasource.url=${QUARKUS_DATASOURCE_URL}
+%prod.quarkus.datasource.username=${QUARKUS_DATASOURCE_USERNAME:}
```

Comment:
Why the colon after USERNAME but not after URL?  optional?

Reply:
The data source URL is required to connect to the DB

Reply:
Others are optional, exactly